dsls/sql/src/main/java/org/apache/beam/dsls/sql/schema/BeamSqlRowCoder.java

```diff
@@ -49,6 +50,8 @@
   private static final DoubleCoder doubleCoder = DoubleCoder.of();
   private static final InstantCoder instantCoder = InstantCoder.of();
   private static final BigDecimalCoder bigDecimalCoder = BigDecimalCoder.of();
+  private static final SerializableCoder<Boolean> booleanCoder =
```

Comment:
make sense especially for deterministcity, would be a problem with SerializableCoder if it's part of GROUP_BY.